src/validateEmail.test.js

```diff
@@ -17,4 +17,138 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   // write more tests here
+  
+  it('should return "false" non-english letters', () => {
+   expect(validateEmail('bąk@doom.com.'))
+    .toBe(false);
+  
+});
+
+it('should return "true" for digits in email', () => {
+  expect(validateEmail('email123@mail123.com123.'))
```

Comment:
The email 'email123@mail123.com123.' is considered invalid because it ends with a dot. The expectation should be `toBeFalsy()` instead of `toBeTruthy()`.